src/app/organizations/registerOrganization/requireAccountsModal/require-accounts-modal.component.html

```diff
@@ -0,0 +1,12 @@
+<h1 mat-dialog-title>User Verification</h1>
+<p>
+  For verification purposes, you are required to have at least two external accounts linked to your Dockstore account. However, it's
+  preferrable to have more than two because Dockstore curators will use that information to verify that your organization request is
+  legitimate.
+</p>
+<p>You have {{ numLinkedAccounts }} account(s) linked.</p>
+<div mat-dialog-actions class="pull-right">
```

Comment:
`Dialog and card action buttons aligned right using fxLayout="row" fxLayoutAlign="end center" fxLayoutGap="1rem"` from https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide